### ISSUE TITLE
How are some of our ideal customers using dashboards?

### ISSUE THREAD

**Issue by annikaschmid** [PostHog/posthog#12195]
**TL;DR**

The main use case I see for "curated dashboards" is to show new users examples for how to use PostHog. There are other (easier?) ways we can achieve this too, e.g. through the new product tour or targeted tutorials.

As soon as a user / organisation is fully onboarded to PostHog, their use cases are too varied to be supported by a "one size fits all" curated dashboard experience.

## Background: 

As part of the [current sprint](https://github.com/PostHog/posthog/issues/12014), the product analytics team is investigating whether providing users with a more curated dashboard experience will make it easier for users to get started with PostHog. As part of this, I wanted to find out whether there are any patterns among existing dashboard setups that will help us identify which dashboard templates might be useful to our users.

## Approach:

I viewed ~30 session recordings of 5 organisations that count as ideal customers. I looked at which and how many dashboards they have set up, what dashboard features they are using and what types of reports they are adding to their dashboards.

## Dashboard features used:

| Organisation | Home dashboard | Curated dashboard by PostHog                        | Dashboard labels | Pinned dashboards | Number of dashboards | Org member count | No. of employees (**[properties.icp___employees](https://docs.google.com/spreadsheets/d/1kHPH1sFHC9_tjLXtw976oo3ToPj7CsmO7pe9ozoc7Ts/edit#gid=143776967)**) | Example recording                                            |
| ------------ | -------------- | --------------------------------------------------- | ---------------- | ----------------- | -------------------- | ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
| 1a           | n              | exists, modified and being used                     | n                | y                 | 3                    | 12               | 140                                                          | [Example recording](https://app.posthog.com/recordings?filters=%7B%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22viewed%20dashboard%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22name%22%3A%22viewed%20dashboard%22%2C%22properties%22%3A%5B%5D%7D%5D%2C%22properties%22%3A%5B%7B%22key%22%3A%22org__name%22%2C%22value%22%3A%22alchemy%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22person%22%7D%5D%2C%22date_from%22%3A%22-30d%22%2C%22date_to%22%3A%22%22%2C%22offset%22%3A0%2C%22session_recording_duration%22%3A%7B%22type%22%3A%22recording%22%2C%22key%22%3A%22duration%22%2C%22value%22%3A60%2C%22operator%22%3A%22gt%22%7D%7D#sessionRecordingId=183a42a027c6e9-0aae50a7fda792-1a525635-1d73c0-183a42a027d2d45) |
| 1b           | y              | doesn't exist anymore                               | n                | y                 | 6                    | 12               | 140                                                          | [Example recording](https://app.posthog.com/recordings?filters=%7B%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22viewed%20dashboard%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22name%22%3A%22viewed%20dashboard%22%2C%22properties%22%3A%5B%5D%7D%5D%2C%22properties%22%3A%5B%7B%22key%22%3A%22org__name%22%2C%22value%22%3A%22alchemy%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22person%22%7D%5D%2C%22date_from%22%3A%22-30d%22%2C%22date_to%22%3A%22%22%2C%22offset%22%3A0%2C%22session_recording_duration%22%3A%7B%22type%22%3A%22recording%22%2C%22key%22%3A%22duration%22%2C%22value%22%3A60%2C%22operator%22%3A%22gt%22%7D%7D#sessionRecordingId=1836bb28a2254a-057b47fca5395c-1a525635-1d73c0-1836bb28a232f37) |
| 2            | y              | still there, looks like it hasn't been used         | n                | y                 | 45+                  | 99               | 180                                                          | [Example recording](https://app.posthog.com/recordings?filters=%7B%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22viewed%20dashboard%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22name%22%3A%22viewed%20dashboard%22%2C%22properties%22%3A%5B%5D%7D%5D%2C%22properties%22%3A%5B%7B%22key%22%3A%22org__name%22%2C%22value%22%3A%22aftership%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22person%22%7D%5D%2C%22date_from%22%3A%22-30d%22%2C%22date_to%22%3A%22%22%2C%22offset%22%3A0%2C%22session_recording_duration%22%3A%7B%22type%22%3A%22recording%22%2C%22key%22%3A%22duration%22%2C%22value%22%3A60%2C%22operator%22%3A%22gt%22%7D%7D#sessionRecordingId=183c158aa207eb-0be5191d829e3-1a525635-13c680-183c158aa2114d8) |
| 3            | y              | doesn't exist anymore                               | n                | y                 | 6                    | 4                | 2020                                                         | [Example recording](https://app.posthog.com/recordings?filters=%7B%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22viewed%20dashboard%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22name%22%3A%22viewed%20dashboard%22%2C%22properties%22%3A%5B%5D%7D%5D%2C%22properties%22%3A%5B%7B%22key%22%3A%22org__name%22%2C%22value%22%3A%22combinator%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22person%22%7D%5D%2C%22date_from%22%3A%22-30d%22%2C%22date_to%22%3A%22%22%2C%22offset%22%3A0%2C%22session_recording_duration%22%3A%7B%22type%22%3A%22recording%22%2C%22key%22%3A%22duration%22%2C%22value%22%3A60%2C%22operator%22%3A%22gt%22%7D%7D#sessionRecordingId=183c2e732fb1f7-00df582b6d531f-1a525635-13c680-183c2e732fecd) |
| 4            | y              | couldn't see it, didn't see full list of dashboards | y                | y                 | 23+                  | 14               | 15                                                           | [Example recording](https://app.posthog.com/recordings?filters=%7B%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22viewed%20dashboard%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22name%22%3A%22viewed%20dashboard%22%2C%22properties%22%3A%5B%5D%7D%5D%2C%22properties%22%3A%5B%7B%22key%22%3A%22org__name%22%2C%22value%22%3A%22bardeen%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22person%22%7D%5D%2C%22date_from%22%3A%222022-09-19%22%2C%22date_to%22%3Anull%2C%22offset%22%3A0%2C%22session_recording_duration%22%3A%7B%22operator%22%3A%22gt%22%2C%22value%22%3A60%2C%22type%22%3A%22recording%22%2C%22key%22%3A%22duration%22%7D%7D#sessionRecordingId=183b38d7f6754d-05aa48d0cd38e1-1a525635-157188-183b38d7f681fa8) |
| 5            | y              | still there, looks like it hasn't been used         | y                | y                 | 4                    | 6                | 5                                                            | [Example recording](https://app.posthog.com/recordings?filters=%7B%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22viewed%20dashboard%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22name%22%3A%22viewed%20dashboard%22%2C%22properties%22%3A%5B%5D%7D%5D%2C%22properties%22%3A%5B%7B%22key%22%3A%22org__name%22%2C%22value%22%3A%22gleam%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22person%22%7D%5D%2C%22date_from%22%3A%222022-09-19%22%2C%22date_to%22%3Anull%2C%22offset%22%3A0%2C%22session_recording_duration%22%3A%7B%22type%22%3A%22recording%22%2C%22key%22%3A%22duration%22%2C%22value%22%3A60%2C%22operator%22%3A%22gt%22%7D%7D#sessionRecordingId=183ab8a1984e2d-0fb63ef3260bac-1a525635-1d73c0-183ab8a198529e9) |

Note: See the spreadsheet below (PostHog internal) if you want to know the names of the companies I looked at.

## Observations around dashboard usage and topics:

- I listed all dashboards and insights on dashboards by organisation I could see. The names of those give us an indication of the content. See the spreadsheet here: https://docs.google.com/spreadsheets/d/1KeNq1k2QAxM_UA1CmpWiToISeXOI-pAvSmSsZx8B_X0/edit#gid=0
- There doesn't seem to be a "one size fits all" approach to how dashboards are set up
- Dashboard topics vary. Some themes identified are high-level product/company dashboards, product-specific dashboards, project/experiment specific dashboards
- Some of the higher-usage companies seem to create dashboards for "everything" (similar to how we use dashboards at PostHog)
- If the list of dashboards is long, the most common way to navigate it is through the search, rather than tags or pinned boards
- The dashboard that is set up by PostHog as a default is not being used much. From the recordings it is unclear if this dashboard was helpful to get started with PostHog
- The dashboard tags are rarely used

## Other observations:

- Organisation 4 has [dashboards](https://app.posthog.com/recordings?filters=%7B%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22viewed%20dashboard%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22name%22%3A%22viewed%20dashboard%22%2C%22properties%22%3A%5B%5D%7D%5D%2C%22properties%22%3A%5B%7B%22key%22%3A%22org__name%22%2C%22value%22%3A%22bardeen%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22person%22%7D%5D%2C%22date_from%22%3A%222022-09-19%22%2C%22date_to%22%3Anull%2C%22offset%22%3A0%2C%22session_recording_duration%22%3A%7B%22operator%22%3A%22gt%22%2C%22value%22%3A60%2C%22type%22%3A%22recording%22%2C%22key%22%3A%22duration%22%7D%7D#sessionRecordingId=183b38d7f6754d-05aa48d0cd38e1-1a525635-157188-183b38d7f681fa8) for their PostHog usage
- Organisation 4 has a dashboard called ""(DO NOT EDIT!) Board Reporting"
- Organisation 2 starts most of their dashboard names by area, e.g. growth, www

## Recommendation:

- From the above findings, there is not one clear theme around a dashboard type or content of a dashboard. Dashboards seem to be created based on the user's preference (e.g. reports with multiple trends vs. one trend per report) and job role/specification (marketing vs. product)
- There is not enough evidence from the observations that would indicate that a dashboard template would help <u>_experienced users_</u> set up new dashboards faster and what this dashboard should be, given that dashboards are so specific
- The current example "App Dashboard" / "Web App Dashboard" PostHog sets up by default has mixed usage. Some companies have deleted it, some haven't touched it and one has amended it. Potentially it useful to get started with PostHog as a <u>_new user_</u>, even if it is just for identifying that the board is not relevant and deleting it. I would recommend we conduct some interviews with recently signed up users to understand how they learn about using PostHog, before considering building more templates as a "quick start"
- Another idea we had was to categorise events through actions, e.g. whether they are part of a conversion funnel. Based on the dashboards I have seen, I am not sure this would be useful to users, because the use cases are so different (marketing vs. product vs. different products)

**Comment by pauldambra** [comment]
This is great analysis!

> I would recommend we conduct some interviews with recently signed up users to understand how they learn about using PostHog, before considering building more templates as a "quick start"

+1

**Comment by mariusandra** [comment]
Great work!

Some thoughts this triggers:

- Did we think too broad when considering a curated AARRR experience at the level of dashboards? Should we instead scope _curation_ down to experiences like "new funnel guided mode"? 
- We could invest in building something nice here, but has anyone really asked for it... and would we get the same or better results with a tooltip and a link "New here? Not sure what to track? Click here for a guide on creating AARRR funnels" when creating a funnel
- We actually have a link to teach people to use funnels here:

<img width="1452" alt="image" src="https://user-images.githubusercontent.com/53387/195195183-a041f652-a682-4328-95e7-a05fee9eff97.png">

- It's a very nice page that teaches people how to use the funnels feature. It could possibly go even deeper on explaining what kind of data you should be paying attention to, or link to further reading on this. Maybe.
- Could we make this empty state more interactive or informative? 
- What about the empty dashboard state? It could use a "apply template" dropdown or another link to documentation. 

<img width="1487" alt="image" src="https://user-images.githubusercontent.com/53387/195195679-67e4b227-dcd0-431f-bc9b-808f064476da.png">

- What if we actually turn this fella into a clippy that literally says "Hi, it looks like you're trying to build a funnel, would you like some help?"

![2022-10-11 22 56 26](https://user-images.githubusercontent.com/53387/195196811-3b435a46-c88d-4d02-897f-5331de10480d.gif)

Overall, looks like we should just make it easier for people to use, edit, fork, build and explore dashboards. Curation isn't really needed here.
